src/main/java/org/opentripplanner/transit/model/timetable/ValidationError.java

```diff
@@ -0,0 +1,11 @@
+package org.opentripplanner.transit.model.timetable;
+
+/**
+ * Details about why a {@link TripTimes} instance is invalid.
+ */
+public record ValidationError(ValidationErrorType type, int stopIndex) {
+  public enum ValidationErrorType {
+    NEGATIVE_DWELL_TIME,
+    NEGATIVE_HOP_TIME,
+  }
```

Comment:
The inner class is repeating the name of the outer class. Two possible solutions, either:
1. Rename to ErrorType/Type/Code or
2. Move out of class

I think `code` is better than `type` here, but feel free to use what you think is best.

Reply:
Done in https://github.com/opentripplanner/OpenTripPlanner/pull/5054/commits/07613c7afa2e0a7b48292ff93dc2d478688f2971